go/libkb/api.go

```diff
@@ -620,12 +620,12 @@ func (a *InternalAPIEngine) PostJSON(arg APIArg) (*APIRes, error) {
 	return a.DoRequest(arg, req)
 }
 
-// PostResp performs a POST request and returns the http response.
+// postResp performs a POST request and returns the http response.
 // The returned response, if non-nil, should have
 // DiscardAndCloseBody() called on it.
-func (a *InternalAPIEngine) PostResp(arg APIArg) (*http.Response, error) {
+func (a *InternalAPIEngine) postResp(arg APIArg, sendJSON bool) (*http.Response, error) {
```

Comment:
Like we already have this panic check:

    PreparePost: sending JSON, but http args exist and they will be ignored. Fix your APIArg.

It feels natural to just keep doing that, but inspect the arg more instead of using the separate bool.